templates/js/sso-tab-with-obo-flow/env/.env.dev

```diff
@@ -0,0 +1,24 @@
+# This file includes environment variables that will be committed to git by default.
```

Comment:
I think we should include .env.local and .env.*.user in the templates. Could you also update the ts template?

Reply:
Updated, thx!